### PR TITLE
added pitchfork charts.

### PR DESCRIPTION
**Pull request by sumitanvekar**
Added initial charts for ExpediaGroup/pitchfork.
fixes issue ExpediaGroup/pitchfork#12 

The test has a retry since if we start the test before pitchfork is up then the test fails. With the retries, even if you start the test immediately, it will give some time for the app to start.

### CODE REVIEW

File: charts/pitchfork/values.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+labels:
+  - app: pitchfork
+  - release: pitchfork
+
+podAnnotations:
+  prometheus.io/scrape: "true"
+  prometheus.io/port: "8081"
+  prometheus.io/path: "/actuator/prometheus"
+
+ingress:
```
Comment (worldtiki) [inline charts/pitchfork/values.yaml:20]: i dont think this is used. we only have a kubernetes service.

File: charts/pitchfork/values.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  enabled : false
+podSecurityContext: {}
+
+securityContext: {}
```
Comment (worldtiki) [inline charts/pitchfork/values.yaml:24]: same for these 2

File: charts/pitchfork/values.yaml (hunk 1 above)
Comment (sumitanvekar) [reply charts/pitchfork/values.yaml]: yeap. it was only used in NOTES.txt so, I cleaned it up.

File: charts/pitchfork/values.yaml (hunk 2 above)
Comment (sumitanvekar) [reply charts/pitchfork/values.yaml]: Removed.

**Pull request merged**